framework-cloud/framework-cloud-api/src/main/java/org/kie/cloud/api/scenario/builder/ClusteredWorkbenchRuntimeSmartRouterTwoKieServersTwoDatabasesScenarioBuilder.java

```diff
@@ -66,15 +66,6 @@
      */
     ClusteredWorkbenchRuntimeSmartRouterTwoKieServersTwoDatabasesScenarioBuilder withLdapSettings(LdapSettings ldapSettings);
 
-    /**
-     * Return setup builder with Business Central user for the maven repository.
-     *
-     * @param user Business Central Maven repo user name.
-     * @param password Business Central Maven repo user password.
-     * @return Builder with configured Business Central Maven repo user.
-     */
-    ClusteredWorkbenchRuntimeSmartRouterTwoKieServersTwoDatabasesScenarioBuilder withBusinessCentralMavenUser(String user, String password);
```

Comment:
This method was not used anywhere. However, I removed it because the user business central maven was deleted.